service/src/main/java/org/cbioportal/service/exception/GeneNotFoundException.java

```diff
@@ -0,0 +1,19 @@
+package org.cbioportal.service.exception;
```

Comment:
creating service layer exceptions here is good. I just wonder if we plan to use exception throwing from the persistence layer too? If so, it might be good to keep all of our custom exception classes together in a lower module. But this is fine for now.